api/operator/v1beta1/vmextra_types.go

```diff
@@ -771,16 +771,16 @@ func (l *License) MaybeAddToArgs(args []string, secretMountDir string) []string
 		return args
 	}
 	if l.Key != nil {
-		args = append(args, fmt.Sprintf("-license=%s", *l.Key))
+		args = append(args, fmt.Sprintf("--license=%s", *l.Key))
 	}
 	if l.KeyRef != nil {
-		args = append(args, fmt.Sprintf("-licenseFile=%s", path.Join(secretMountDir, l.KeyRef.Name, l.KeyRef.Key)))
+		args = append(args, fmt.Sprintf("--licenseFile=%s", path.Join(secretMountDir, l.KeyRef.Name, l.KeyRef.Key)))
 	}
 	if l.ForceOffline != nil {
-		args = append(args, fmt.Sprintf("-license.forceOffline=%v", *l.ForceOffline))
+		args = append(args, fmt.Sprintf("--license.forceOffline=%v", *l.ForceOffline))
```

Comment:
This hidden change will trigger restart of all applications that uses license flag.

 Operator provides guarantees,  that version update will never trigger restart of VMSingle and VMCluster components. Please rollback this change. And make sure, that nothing else changed in the same manner.